tests/topotests/multicast_pim6_bsm_topo1/test_mcast_pim6_bsm_01.py

```diff
@@ -0,0 +1,1777 @@
+    not processed
+4.2 Verify group to RP info updated correctly in FRR node, after shut and
+    no-shut of BSM enable interfaces
+5. Verify static RP is preferred over BSR
+6.1 Verify adding/deleting the group to rp mapping and RP priority
+    multiple times
+6.2 Verify RP and (*,G) detail after PIM process restart on FRR node
+7.1 Verify BSM timeout on FRR1
+7.2 Verify RP state in FRR1 after Bootstrap timer expiry
+8.1 Verify upstream interfaces(IIF) and join state are updated properly
+    after BSM received for FRR
+8.2 Verify IIF and OIL in "show ip pim state" updated properly after
+    BSM received
+"""
+
+import os
+import sys
+import time
+import pytest
+
+# Save the Current Working Directory to find configuration files.
+CWD = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(CWD, "../"))
+sys.path.append(os.path.join(CWD, "../lib/"))
+
+# Required to instantiate the topology builder class.
+
+# pylint: disable=C0413
+# Import topogen and topotest helpers
+from lib.topogen import Topogen, get_topogen
+from re import search as re_search
+from re import findall as findall
+
+from lib.common_config import (
+    start_topology,
+    write_test_header,
+    write_test_footer,
+    step,
+    addKernelRoute,
+    create_static_routes,
+    stop_router,
+    start_router,
+    shutdown_bringup_interface,
+    kill_router_daemons,
+    start_router_daemons,
+    reset_config_on_routers,
+    apply_raw_config,
+    run_frr_cmd,
+    required_linux_kernel_version,
+    verify_rib,
+    scapy_send_raw_packet,
+)
+
+from lib.pim import (
+    create_pim_config,
+    add_rp_interfaces_and_pim_config,
+    reconfig_interfaces,
+    scapy_send_bsr_raw_packet,
+    find_rp_from_bsrp_info,
+    verify_pim_grp_rp_source,
+    verify_pim_bsr,
+    verify_mroutes,
+    verify_join_state_and_timer,
+    verify_pim_state,
+    verify_upstream_iif,
+    verify_igmp_groups,
+    verify_pim_upstream_rpf,
+    enable_disable_pim_unicast_bsm,
+    enable_disable_pim_bsm,
+    clear_mroute,
+    clear_pim_interface_traffic,
+    get_pim6_interface_traffic,
+    McastTesterHelper,
+    verify_pim_neighbors,
+    create_mld_config,
+)
+from lib.topolog import logger
+from lib.topojson import build_config_from_json
+
+
+pytestmark = [pytest.mark.pimd, pytest.mark.staticd]
+
+TOPOLOGY = """
+
+      b1_____
+             |
+             |
+      s1-----f1-----i1-----l1----r1
+             |
+       ______|
+      b2
+
+      b1 - BSR 1
+      b2 - BSR 2
+      s1 - Source
+      f1 - FHR
+      i1 - Intermediate Router (also RP)
+      r1 - Receiver
+
+"""
+# Global variables
+GROUP_RANGE = "ff00::/8"
+GROUP_RANGE_1 = [
+    "ffaa::1/128",
+    "ffaa::2/128",
+    "ffaa::3/128",
+    "ffaa::4/128",
+    "ffaa::5/128",
+]
+MLD_JOIN_RANGE_1 = ["ffaa::1", "ffaa::2", "ffaa::3", "ffaa::4", "ffaa::5"]
+
+CRP_ADDR = "3000::1/128"
+BSRP_ADDR = "5555::1/128"
+CRP_ADDR = "3000::1/128"
```

Comment:
CRP_ADDR is defined twice. Please remove one.

Reply:
Removed.